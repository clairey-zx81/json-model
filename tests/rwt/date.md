- "format" keyword seems to be ignored by schema checker
