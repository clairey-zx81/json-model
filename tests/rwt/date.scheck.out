date.0.true.json: True
date.1.true.json: True
date.0.false.json: False
date.1.false.json: False
date.2.false.json: False
